splink/connected_components.py

```diff
@@ -475,12 +475,7 @@ def solve_connected_components(
             prev_representatives_table.physical_name
         )
 
-        # To allow debug mode to work with our recursive loop, add
-        # the iteration number as a suffix
-        if linker.debug_mode:
-            repr_name = "__splink__df_representatives"
-        else:
-            repr_name = f"__splink__df_representatives_{iteration}"
+        repr_name = f"__splink__df_representatives_{iteration}"
```

Comment:
Have you checked if this works w/ debug mode?

Reply:
Yes - it does work